### ISSUE TITLE
Timex.Date.shift\2 Incorrect

### ISSUE THREAD

**Issue by stavro** [bitwalker/timex#81]
`Timex.Date.shift/2` with interval argument `mins: <integer>` returns an incorrect result when given a DateTime not in UTC time.

```elixir
{{2015, 09, 24}, {10, 0, 0}}
|> Timex.Date.from("America/Los_Angeles")
|> Timex.Date.shift(mins: 45)
|> Timex.DateFormat.format!("{ISO}")
# => "2015-09-24T10:45:00+0000"
```

The problem is that `Timex.Date.shift/2` always assumes the input is it UTC, and thus treats date and time components as UTC components erroneously, clearing out the Time Zone.

Reference: https://github.com/bitwalker/timex/blob/master/lib/date/date.ex#L1153-L1161

**Comment by bitwalker** [comment]
Hey @stavro, thanks for reporting this! I've applied a fix and pushed a new release (0.19.5) to hex - and fixed the tests which should've caught this :/

**Comment by stavro** [comment]
Thanks for the speedy fix!
